index.html

```diff
@@ -2810,6 +2813,84 @@ <h2 class="icon-title">
           </div>
         </section>
       </section>
+      <section id="solid-color-icons">
+        <h2>
+          Solid color icons
+        </h2>
+        <p>
+          Some platforms enforce that icons be displayed with a single color
+          or gradient, where only the transparency of the icon can be
+          controlled. As web applications should work across multiple
+          platforms, it it possible to indicate that an icon can have a
+          user-agent-specified color applied by adding the <a>monochrome</a>
+          purpose. This allows the platform to ensure that the icon looks well
+          integrated with the platform, and even apply different colors and
+          padding in different places throughout the platform.
+        </p>
+        <p>
+          When processing a <a>monochrome</a> icon, the user agent MUST NOT use
+          the red component, green component, or blue component of a pixel. If
+          it has alpha equal to zero, the user agent SHOULD NOT display it.
+          If it the alpha component is greater than zero, the user agent SHOULD
+          display it with any tint.
```

Comment:
Does the OS or the UA display these?

Reply:
Here I'm trying to indicate that r,g,b should be ignored and only alpha should be used. Alpha 0 is transparent so it shouldn't show up. I expect the processing to strip out/replace rgb and leave alpha alone, then the OS will display the icon after.